ctrl_frames_flood/main.go

```diff
@@ -0,0 +1,197 @@
+package main
```

Comment:
Please move this script to `tools/` directory. In the future, I plan to move all other tools/service to this directory.

Reply:
yes fixed